src/tasks/ruuvi_task_adc.h

```diff
@@ -177,5 +177,18 @@ rd_status_t rt_adc_vdd_sample (void);
  */
 rd_status_t rt_adc_vdd_get (float * const vdd);
 
+/**
+ * @brief Get VDD Sample from selected ADC handle
+ *
+ * This function initializes ADC, reads VDD sample from selected handle
+ * and returns the sample as the float value.
+ *
+ * @param[in] config Configuration of ADC.
+ * @param[in] handle Handle to ADC, i.e. ADC pin.
+ * @retval vdd VDD voltage in volts.
```

Comment:
Fix the retail comment

Reply:
Rephrased as `@retval error code from stack on error.`